lms/static/scripts/frontend_apps/components/ErrorDialogApp.tsx

```diff
@@ -36,6 +36,10 @@ export default function ErrorDialogApp() {
       title = 'Hypothesis license activated';
       displaySupportLink = false;
       break;
+    case 'vitalsource_student_pay_license_launch_instructor':
+      title = 'This button allows students to acquire licenses';
```

Comment:
Why do we mention the concept of a "button" as the title of the modal? What button is it referring to?

Reply:
Good point: the way you access H via VS to see this message is via this "Launch Courseware" button:

![Screenshot from 2024-05-06 12-37-23](https://github.com/hypothesis/lms/assets/1433832/c85ef724-1e00-4b91-b62d-97fd5b5104de)

https://hypothesis.instructure.com/courses/638/external_tools/898